src/drafts/MarkdownEditor/MarkdownEditor.test.tsx

```diff
@@ -800,19 +798,8 @@ describe('MarkdownEditor', () => {
         const link = await waitFor(() => within(getPreview()).getByText('Link'))
 
         await user.click(link)
-        // Note: navigation is not implemented in JSDOM and will log out an
-        // error when clicking the link above. The spy here captures this error
-        // and will assert that it is called only once, otherwise another error
-        // in this test has occurred
-        expect(spy).toHaveBeenCalledTimes(1)
```

Comment:
This test was previously asserting the buggy behavior - now that we're not navigating the page (the call to window.open is mocked), we don't see any error.